articles/azure-functions/functions-scale.md

```diff
@@ -29,7 +29,7 @@ App Service plan allows you to take advantage of dedicated infrastructure, which
 Feature support falls into the following two categories:
 
 * _Generally available (GA)_: fully supported and approved for production use.
-* _Preview_: not yet fully supported and approved for production use.
+* _Preview_: not yet fully supported nor approved for production use.
```

Comment:
Made it sound like Preview means the feature is not yet fully supported but is approved for production use